dev/bots/test/codesign.dart

```diff
@@ -0,0 +1,309 @@
+          '-type',
+          'f',],
+        stdout: allFilesStdout,
+      );
+      commandList.add(findCmd);
+      for (final String expectedFile in allExpectedFiles) {
+        commandList.add(
+          FakeCommand(
+            command: <String>[
+              'file',
+              '--mime-type',
+              '-b',
+              expectedFile,
+            ],
+            stdout: 'application/x-mach-binary',
+          )
+        );
+      }
+      final ProcessManager processManager = FakeProcessManager.list(commandList);
+      final List<String> foundFiles = await findBinaryPaths('$flutterRoot/bin/cache', processManager: processManager);
+      expect(foundFiles, allExpectedFiles);
+    });
+
+    test('Empty file list', () async {
+      final List<FakeCommand> commandList = <FakeCommand>[];
+      const FakeCommand findCmd = FakeCommand(
+        command: <String>[
+          'find',
+          '$flutterRoot/bin/cache',
+          '-type',
+          'f',],
+      );
+      commandList.add(findCmd);
+      final ProcessManager processManager = FakeProcessManager.list(commandList);
+      final List<String> foundFiles = await findBinaryPaths('$flutterRoot/bin/cache', processManager: processManager);
+      expect(foundFiles, <String>[]);
+  });
+
+  group('isBinary', () {
+    test('isTrue', () async {
+      final List<FakeCommand> commandList = <FakeCommand>[];
+      const String fileToCheck = '/a/b/c/one.zip';
+      const FakeCommand findCmd = FakeCommand(
+        command: <String>[
+          'file',
+          '--mime-type',
+          '-b',
+          fileToCheck,
+        ],
+        stdout: 'application/x-mach-binary',
+      );
+      commandList.add(findCmd);
+      final ProcessManager processManager = FakeProcessManager.list(commandList);
+      final bool result = await isBinary(fileToCheck, processManager: processManager);
+      expect(result, isTrue);
+    });
+
+    test('isFalse', () async {
+      final List<FakeCommand> commandList = <FakeCommand>[];
+      const String fileToCheck = '/a/b/c/one.zip';
+      const FakeCommand findCmd = FakeCommand(
+        command: <String>[
+          'file',
+          '--mime-type',
+          '-b',
+          fileToCheck,
+        ],
+        stdout: 'text/xml',
+      );
+      commandList.add(findCmd);
+      final ProcessManager processManager = FakeProcessManager.list(commandList);
+      final bool result = await isBinary(fileToCheck, processManager: processManager);
+      expect(result, isFalse);
+    });
+  });
+
+  group('hasExpectedEntitlements', () {
+     test('expected entitlements', () async {
+       final List<FakeCommand> commandList = <FakeCommand>[];
+       const String fileToCheck = '/a/b/c/one.zip';
+       const FakeCommand codesignCmd = FakeCommand(
+         command: <String>[
+           'codesign',
+           '--display',
+           '--entitlements',
+           ':-',
+           fileToCheck,
+         ],
+         exitCode: 0,
```

Comment:
analysis failing here:

```
   info • The value of the argument is redundant because it matches the default value • dev/bots/test/codesign.dart:169:20 • avoid_redundant_argument_values
```

Reply:
Removed.